fern/docs/pages/welcome/quickstart.mdx

```diff
@@ -1,53 +1,63 @@
-### Install
+In this quickstart, we'll install Fern, initialize a sample API, and generate a TypeScript Node.js SDK. The output files will be downloaded to your local file system.
 
-```bash
-npm install -g fern-api
-```
-
-### Already have an OpenAPI spec?
-
-Import your OpenAPI spec into Fern [here](/api-definition/open-api-specification/openapi/import).
+We'll use a [**Fern Definition**](/api-definition/fern-definition/overview) to describe our endpoints, types, and errors. Alternatively, you can use an [OpenAPI Specification](/openapi/import).
```

Comment:
- lower case 'specification' per recent discussion
- second link should be to `(/api-definition/openapi/import)`